torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -86,7 +87,8 @@ bool isSimpleMap(Node* node) {
       "aten::add(Tensor self, Scalar other, Scalar alpha) -> Tensor",
       "aten::sub(Tensor self, Scalar other, Scalar alpha) -> Tensor",
       "aten::mul(Tensor self, Scalar other) -> Tensor",
-      "aten::div(Tensor self, Scalar other) -> Tensor",
+      "aten::div(Tensor self, Scalar other, str rounding_mode) -> Tensor",
```

Comment:
I don't think we need to change this if this PR keeps the signature without rounding_mode?